gtsam/geometry/tests/testRot3.cpp

```diff
@@ -661,6 +661,21 @@ TEST(Rot3, ClosestTo) {
   EXPECT(assert_equal(expected, actual.matrix(), 1e-6));
 }
 
+/* ************************************************************************* */
+TEST(Rot3, angle) {
+  Rot3 R1 = Rot3::Ypr(0, 0, 0);
```

Comment:
this is identity. Which is good, as your test is almost good for static method. Just kill R1, rename R2 to R1. Or, better, just test angle for all rotations defined at top of this file...